README.md

```diff
@@ -1,4 +1,4 @@
-# watsonwork-weather
+# watsonwork-weather-intent-based
```

Comment:
Would it make sense to separate this project into two different repos: one for action responses and one for intent responses?